crates/detect-targets/Cargo.toml

```diff
@@ -10,7 +10,10 @@ edition = "2021"
 license = "Apache-2.0 OR MIT"
 
 [dependencies]
+futures-util = "0.3.28"
 tokio = { version = "1.28.2", features = ["rt", "process", "sync"], default-features = false }
+tracing = "0.1.37"
+tracing-subscriber = { version = "0.3.17", features = ["fmt"], default-features = false }
```

Comment:
You can make tracing-subscriber an optional dependency, only enabled if the binary is built.